allensdk/internal/api/behavior_ophys_api.py

```diff
@@ -33,10 +33,21 @@ def get_sync_data(self):
     def get_stimulus_timestamps(self):
         return self.get_sync_data()['stimulus_frames']
 
-
     @memoize
     def get_ophys_timestamps(self):
-        return self.get_sync_data()['ophys_frames']
+
+        ophys_timestamps = self.get_sync_data()['ophys_frames']
+        dff_traces = self.get_raw_dff_data()
+        number_of_cells, number_of_dff_frames = dff_traces.shape
+        num_of_timestamps = len(ophys_timestamps)
+        if number_of_dff_frames < num_of_timestamps:
+            ophys_timestamps = ophys_timestamps[:number_of_dff_frames]
+        elif number_of_dff_frames == num_of_timestamps:
```

Comment:
no big deal but this branch could be replaced with just the check on >

Reply:
Yeah, I wanted to be explicit for the anyone reading the logic of the code